packages/functional-tests/tests/misc/forceAuth.spec.ts

```diff
@@ -19,6 +19,11 @@ test.describe('force auth', () => {
     credentials,
     pages: { login, resetPassword, forceAuth },
   }) => {
+    // TODO: Skipping this test because React pages does not prefill email for reset password
+    // (We don't have forceAuth at all)
+    const config = await login.getConfig();
+    test.skip(config.showReactApp.resetPasswordRoutes === true);
+    test.skip(true);
```

Comment:
Probably fine to always skip this test.

Reply:
Ah yeah I'll remove that first skip against the config.